src/PointwiseFunctions/Hydro/MassFlux.hpp

```diff
@@ -0,0 +1,61 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "PointwiseFunctions/GeneralRelativity/TagsDeclarations.hpp" // IWYU pragma: keep
+#include "PointwiseFunctions/Hydro/TagsDeclarations.hpp" // IWYU pragma: keep
+#include "Utilities/TMPL.hpp"
+
+// IWYU pragma: no_forward_declare gr::Tags::Lapse
+// IWYU pragma: no_forward_declare gr::Tags::Shift
+// IWYU pragma: no_forward_declare gr::Tags::SqrtDetSpatialMetric
+// IWYU pragma: no_forward_declare hydro::Tags::LorentzFactor
+// IWYU pragma: no_forward_declare hydro::Tags::RestMassDensity
+// IWYU pragma: no_forward_declare hydro::Tags::SpatialVelocity
+
+namespace hydro {
+/// Computes the vector \f$J^i\f$ in \f$\dot{M} = -\int J^i s_i d^2S\f$,
+/// representing the mass flux through a surface with normal \f$s_i\f$.
+/// Note that the integral is understood
+/// as a flat-space integral: all metric factors are included in \f$J^i\f$.
+/// In particular, if the integral is done over a Strahlkorper, the
+/// euclidean_area_element of the Strahlkorper should be used, and \f$s_i\f$ is
```

Comment:
Can you put ticks around `euclidean_area_element` and include the namespace so Doxygen makes it link correctly?